starter-code/Bookshelf/app/src/main/java/ly/generalassemb/drewmahrt/bookshelf/MainActivity.java

```diff
@@ -52,11 +58,37 @@ public long getItemId(int position) {
             @Override
             public View getView(int position, View convertView, ViewGroup parent) {
                 //TODO: Update the view
+                View cView = convertView;
```

Comment:
Using `cView` here is not wrong, but you can actually just use `convertView` without making a new variable if you want. If `convertView` is null, you can assign the newly inflated view directly to it, rather than having to use a new local variable.